Update index.html.md

DalwynWasTaken opened the pull request:
Added version to example metadata and added advice to make versions match when uploading Linked javadocs to the javadocs and IRegister to IRegister, the previous version looked really bad Added setAliases to command trigger

Not sure what's going on with the chat trigger since you can't see it on the published website (chattriggers.com/slate) but Added a push to not use setContains when trying to pickup very specific messages along with the reason why

I think this file is good, it covers the basics to make a module and I don't think it needs more for now.  Will slowly work my way to the other files.

camnwalter commented on source/index.html.md:
```diff
@@ -128,7 +130,7 @@ message being sent. Let's start with one of the simplest triggers: WorldLoad. In
 the trigger name as the first argument (case-insensitive), and the function to
 run as the second argument. You can see a complete list of these triggers on our
-[javadocs, under `IRegister`](https://www.chattriggers.com/javadocs/com/chattriggers/ctjs/engine/IRegister.html).
+[javadocs](https://www.chattriggers.com/javadocs/), under [`IRegister`](https://www.chattriggers.com/javadocs/com/chattriggers/ctjs/engine/IRegister.html).
```
That link is dead, it should be `https://chattriggers.com/javadocs/-chat-triggers/com.chattriggers.ctjs.engine/-i-register/index.html`

DalwynWasTaken committed:
Update index.html.md

Added version to example metadata and added advice to make versions match when uploading
Linked javadocs to the javadocs and IRegister to IRegister, the previous version looked really bad
Added setAliases to command trigger

Not sure what's going on with the chat trigger since you can't see it on the published website (chattriggers.com/slate) but
Added a push to not use setContains when trying to pickup very specific messages along with the reason why

I think this file is good, it covers the basics to make a module and I don't think it needs more for now. 
Will slowly work my way to the other files

DalwynWasTaken commented:
Done, also you need to redeploy the slate since the infamous `contains` has long been patched but from what I can tell not been deployed to the website
